### DISCUSSION TITLE
Why postcss-nested is enabled before Tailwind, but Vite drops Warning in terminal?

### DISCUSSION THREAD

**Discussion by roman-hrynevych-knm** [tailwindlabs/tailwindcss#9951] [Help]
```
import nested from 'postcss-nested';
import tailwind from 'tailwindcss';

postcss: {
      plugins: [
        pcssImport,
        nested,
        tailwind,
        // other plugins
      ],
    },
```

Nested is over tailwind but warning is in terminal(
![image](https://user-images.githubusercontent.com/117681903/204336655-b3806862-17dd-4337-a439-2e2c71eecfb3.png)

**Comment by wongjn** [comment]
As per the warning, [check the documentation on nesting](https://tailwindcss.com/docs/using-with-preprocessors#nesting). In particular, using the `tailwindcss/nesting` as a wrapper for `postcss-nested`.

**Reply by hrynevychroman** [reply]
Yea, i checked it, but it doesn't helped too(
<img width="698" alt="image" src="https://user-images.githubusercontent.com/82209198/204751228-efdfaaf2-55c2-48ad-b573-f4c1b1f04cd5.png">

**Reply by hrynevychroman** [reply]
this is full screenshot of postcss.config.cjs
<img width="613" alt="image" src="https://user-images.githubusercontent.com/82209198/204751675-b99818ca-cda8-4099-825a-c06594ad2984.png">
